scripts/feature-flags.js

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+const ajv = require('ajv')();
+const Promise = require('../promise');
+const redis = require('../redis')({
+  enabled: true,
+  host: process.env.REDIS_HOST || '127.0.0.1',
+  port: process.env.REDIS_PORT || 6379,
+  prefix: 'featureFlags:',
+  maxConnections: 2,
+  maxPending: 1,
+  minConnections: 1,
+}, {
+  error () {},
+  info () {},
+  warn () {}
+});
+const schema = require('../feature-flags/schema.json');
+const validate = ajv.compile(schema);
+
+const COMMANDS = {
+  clear,
+  merge,
+  read,
+  write,
+  revert,
+};
+const KEYS = {
+  current: 'current',
+  previous: 'previous'
+};
+
+const { argv } = process;
+
+main().then(() => redis.close());
+
+async function main () {
+  try {
+    if (argv.length !== 3) {
+      usageError();
+    }
+
+    const command = COMMANDS[argv[2]];
+    if (! command) {
+      usageError();
+    }
+
+    const result = await command();
+    if (result) {
+      console.log(result);
+    }
+  } catch (error) {
+    console.error(error.message);
+    process.exit(1);
+  }
+}
+
+function usageError () {
+  const scriptName = argv[1].substr(argv[1].indexOf('/scripts/') + 1);
+  throw new Error([
+    'Usage:',
+    `${scriptName} read - Read feature flags to stdout`,
+    `${scriptName} write - Write feature flags from stdin, clobbering existing flags`,
+    `${scriptName} merge - Merge feature flags from stdin (new properties take precedence)`,
+    `${scriptName} clear - Delete all feature flags`,
+    `${scriptName} revert - Undo the last write, merge, clear or revert`,
+  ].join('\n'));
+}
+
+async function read () {
+  const flags = await redis.get(KEYS.current);
+  if (flags) {
+    // Parse then stringify for pretty printing
+    return JSON.stringify(JSON.parse(flags), null, '  ');
```

Comment:
It could be useful to print out something along the lines of `no feature flags set` if none are set, currently it prints nothing which made me wonder "is it working?"

```bash
➜  fxa-shared git:(pb/41-feature-flags) node scripts/feature-flags.js read                          git:(pb/41-feature-flags|)
➜  fxa-shared git:(pb/41-feature-flags)
```